lib/options/sort-order.js

```diff
@@ -283,7 +283,44 @@ module.exports = {
             propertyName = null;
             // Look for includes:
             if (node.get(i).is('include')) {
-                propertyName = '$include';
+                // Divide `include` into `$extend`,
+                // mixins with specific name (e. g. `$include breakpoint`),
+                // and the rest — `$include`.
+                var includeType;
+                var mixinName;
+
+                // SASS and SCSS both supports `@extend` and `@include`,
+                // but SASS also supports `+mixin-name`
+                if (_this.syntax === 'sass' || _this.syntax === 'scss') {
+                    if (_this.syntax === 'sass' && node.get(i).get(0).content === '+') {
```

Comment:
There are three syntax checks in this block. Maybe we can rearrange them a bit. For example:
```js
// We call `node.get(i)` a lot, so we can save it:
var include = node.get(i);
// There is already a local var `syntax` (see line 51), so we can use it.
if (syntax === 'less') {
  // ...
} else if (syntax === 'sass' && include.get(0).content === '+') {
  // ....
} else {
  // Since we already know that syntax is neither `less` nor `css` there is no need for one more `if`
  // ...
}
```
What do you think?